pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -33,16 +37,17 @@ import (
 var log = logf.Log.WithName("controller_wildflyserver")
 
 const (
-	httpApplicationPort         int32 = 8080
-	httpManagementPort          int32 = 9990
-	standaloneServerDataDirPath       = "/wildfly/standalone/data"
+	httpApplicationPort           int32 = 8080
+	httpManagementPort            int32 = 9990
+	recoveryPort                  int32 = 4712
+	standaloneServerDataDirPath         = "/wildfly/standalone/data"       // data directory where runtime data is saved
+	wftcDataDirName                     = "ejb-xa-recovery"                // data directory where WFTC stores transaction runtime data
+	wflyMgmtTxnRecoveryUserName         = "transaction.recovery.scaledown" // username for recovery to run mgmt operations
+	labelWildflyOperatorInService       = "wildfly.operator.in.service"    // label used to remove a pod from receiving load from service during transaction recovery
```

Comment:
We should follow other labels convention with something like `wildfly.org/service-state` 
The label is stamped on the pod, I'm not sure that mentioning the operator in the name is meaningful..

Reply:
sure, I see. Could be the name be like `wildfly.org/operated-by-service` or similar?